tests/data/baseballmetrics/CMakeLists.txt

```diff
@@ -33,9 +33,6 @@ else()
     set(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR})
 endif()
 
-# ensure that the library is a .so on mac
-set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
```

Comment:
This was nonstandard thing I'd copied over from LightGBM. Removing it simplifies this example further, and makes it look a bit more like what I'd expect most projects to look like.